mypy/errorcodes.py

```diff
@@ -53,7 +57,9 @@ def __str__(self) -> str:
 ASSIGNMENT: Final = ErrorCode(
     "assignment", "Check that assigned value is compatible with target", "General"
 )
-TYPE_ARG: Final = ErrorCode("type-arg", "Check that generic type arguments are present", "General")
+TYPE_ARG: Final[ErrorCode] = ErrorCode(
+    "type-arg", "Check that generic type arguments are present", "General"
+)
 TYPE_VAR: Final = ErrorCode("type-var", "Check that type variable values are valid", "General")
```

Comment:
Why some consts are `: Final = ErrorCode()` and some are `: Final[ErrorCode] = ErrorCode()`?

Reply:
mypyc bug. It doesn't recognize just `Final` for some reason, it has to be told. The bug only shows up when that error code is used inside the `ErrorMessage` tuple, so to keep a minimal diff I'm only migrating the error codes that have been used in these PRs.

Eventually as I make more PRs they will all become `Final[ErrorCode]`